src/Microsoft.DotNet.Archive/ConsoleProgressReport.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+
+namespace Microsoft.DotNet.Archive
+{
+    public class ConsoleProgressReport : IProgress<ProgressReport>
+    {
+        string currentPhase;
```

Comment:
Sorry this was all my standalone (prototype rewrite) code and first contribution to CLI.  Can you point me to your guidelines and I'll apply formatting/refactoring as appropriate after we review the functionality?